README.md

```diff
@@ -28,6 +20,16 @@ Another option for Redis and MongoDB could be charts provided by Bitnami:
 
 *Important Note regarding MongoDB:* This helm chart enables the PodDisruptionBudget for MongoDB with an arbiter replica-count of 1.  If you intend to perform system maintenance on the node where the MongoDB pod is running and this maintenance requires for the node to be drained, this action will be prevented due the replica count being 1.  Increase the replica count in the helm chart deployment to a minimum of 2 to remedy this issue.
 
+Another option for Redis and MongoDB, if you want to get started quickly is to use mongo.yaml and redis.yaml manifests located inside deploy/dependencies folder in this repository.
+**Please note that these provided manifests must not ever be used in production and for anything but a quick start evaluation only, use external DBs or Official Helm charts for MongoDB and Redis in any other case.**
+We provide these manifests so you can quickly have Tyk running however they are not meant for long term storage of data for example.
+
+	kubectl create namespace tyk
+	kubectl apply -f deploy/dependencies/mongo.yaml -n tyk
+	kubectl apply -f deploy/dependencies/redis.yaml -n tyk
```

Comment:
I'm thinking it's better to remove these deps entirely from the repo, and link to using our simple mongo/redis helm charts.

Reply:
Agree, dependencies are now removed.